# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleMapKey.h
# Commit: b8ecf23519c9
# Full Hash: b8ecf23519c9ab5180d08f00c712674bfe968dac
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-21 21:39:46
# Regressor Bug: 1742437
# File Overlap Count: 5
# Description:
#   Bug 1742437 - re-scope all generic module/script loader files under js/loader; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138133
# ==============================================================================

diff -r 85fd9fc9cffa -r b8ecf23519c9 js/loader/ModuleMapKey.h
--- a/js/loader/ModuleMapKey.h	Mon Feb 21 15:39:08 2022 +0000
+++ b/js/loader/ModuleMapKey.h	Mon Feb 21 15:39:09 2022 +0000
@@ -8,7 +8,7 @@
 #include "nsURIHashKey.h"
 #include "nsIGlobalObject.h"
 
-namespace mozilla::dom {
+namespace JS::loader {
 
 class ModuleMapKey : public nsURIHashKey {
  public:
@@ -42,6 +42,6 @@
   ImplCycleCollectionTraverse(aCallback, aField.mWebExtGlobal, aName, aFlags);
 }
 
-}  // namespace mozilla::dom
+}  // namespace JS::loader
 
 #endif